cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -56,6 +76,7 @@ func New(log logrus.FieldLogger, db store.Storer, testMode bool) *API {
 	r.Post("/dmsg-discovery/entry/{pk}", api.setEntry())
 	r.Get("/dmsg-discovery/available_servers", api.getAvailableServers())
 	r.Get("/dmsg-discovery/health", api.health())
+	r.Get("/dmsg-discovery/service-health", api.serviceHealth)
```

Comment:
Should we be consistent with the other API's in services and just use `/health`? Why are we introducing a second health endpoint? Any reason not to merge `/health` and `/service-health`?

Reply:
> Should we be consistent with the other API's in services and just use /health? 

Maybe you are right. I will change the name. 

> Why are we introducing a second health endpoint?

```
// MakeHealthHandler returns a HTTP handler that displays component status(es).
// The endpoint returns a content type of text/plain with each component's
// health on a new line. The format of a line is as follows:
//	<componentName>: <HTTPStatusCode> <statusMessage>
// One can request the endpoint to return the health of a single component only
// via the following path:
//	/<expectedBase>/<componentName
```
The old endpoint returns data in weird data format and it is no an easy way to merge them. I do not know if somebody uses it. I guess we need to left them both to keep backward compatibility.